cloudstack/python/tar/slipstream_cloudstack/CloudStackClientCloud.py

```diff
@@ -140,9 +144,24 @@ def _start_image_on_cloudstack(self, user_info, node_instance, vm_name):
                   id=instance.id)
         return vm
 
+    @override
     def list_instances(self):
         return self._thread_local.driver.list_nodes()
 
+    @override
+    def _create_allow_all_security_group(self):
+        sg_name = NodeDecorator.SECURITY_GROUP_ALLOW_ALL_NAME
+        sg_desc = NodeDecorator.SECURITY_GROUP_ALLOW_ALL_DESCRIPTION
+        driver = self._thread_local.driver
+
+        if any([sg.get('name') == sg_name for sg in driver.ex_list_security_groups()]):
+            return
+
+        sg = driver.ex_create_security_group(sg_name, description=sg_desc)
+        driver.ex_authorize_security_group_ingress(sg_name, 'tcp', '0.0.0.0/0', 0, 65535)
+        driver.ex_authorize_security_group_ingress(sg_name, 'udp', '0.0.0.0/0', 0, 65535)
```

Comment:
I see that in OpenStack you are starting from 1. https://github.com/slipstream/SlipStreamConnectors/pull/51/files#diff-7c47d1faa03b4c03eddd7d9ff2deb126R210